<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QWidget" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Image to GIF</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0">
    <property name="verticalSpacing">
     <number>-1</number>
    </property>
    <item row="4" column="1" colspan="2">
     <widget class="QSpinBox" name="scale_spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QSpinBox" name="duration_spinBox">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>250</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Output Path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Images Path</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="outputPath_lineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="export_btn">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Input Type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="browse_outputPath_btn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="inputPath_lineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Scale (%)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="type_ComboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1" rowspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Duration (ms)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="browse_inputPath_btn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QLabel" name="status_label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(150, 150, 150);</string>
      </property>
      <property name="text">
       <string>staus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
